beacon-chain/core/electra/consolidations.go

```diff
@@ -78,7 +77,7 @@ func ProcessPendingConsolidations(ctx context.Context, st state.BeaconState) err
 		if err != nil {
 			return err
 		}
-		b := min(validatorBalance, helpers.ValidatorMaxEffectiveBalance(sourceValidator))
+		b := min(validatorBalance, sourceValidator.EffectiveBalance())
```

Comment:
All the changes here are related to: https://github.com/ethereum/consensus-specs/pull/4020
Also updated spec test above